config_fr.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<module>
+    <name>autoupgrade</name>
+    <displayName><![CDATA[1-Click Upgrade]]></displayName>
+    <version><![CDATA[4.9.0]]></version>
```

Comment:
Probably a detail but maybe it would be better to update this version as well?

Reply:
Removed this file.

Reply:
This file could actually be removed